k8s/nginx/container/Dockerfile

```diff
@@ -7,5 +7,5 @@ RUN apt-get update \
     && apt-get clean
 COPY nginx.conf.template /etc/nginx/nginx.conf
 COPY nginx_entrypoint.bash /
-EXPOSE 80 443 27017
+EXPOSE 80 81 443 444 27017
```

Comment:
That's a good question.
Currently we run the aiohttp on `9985` and the flask server at `9984`.

```
netstat -nalp | grep LIST
tcp        0      0 0.0.0.0:9984            0.0.0.0:*               LISTEN      54/python3.6    
tcp        0      0 0.0.0.0:9985            0.0.0.0:*               LISTEN      55/python3.6  
```

We can have one public port exposed to the outside world and filter by URL and forward to different bdb ports internally. For example, all URLs with `/stream` can be forwarded to the backend at port `9985`.

I decided against that because:

1.  `wss` is treated as a `stream` in NGINX, and hence it would be a non-trivial change when we move from `ws` to `wss`.

2.  I wanted to map the deployment to be similar to the code-base underneath. This will ensure that there are no surprises when users with no deployment experience have a look at it.